Packs/DeveloperTools/Scripts/FetchFromInstance/FetchFromInstance.py

```diff
@@ -0,0 +1,76 @@
+import re
+
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+
+
+def get_instance_name(args):
+    brand_name = args.get("brand_name")
```

Comment:
Move this to be before the `if brand_name`.
You're not using it yes, so no need for the extra  work and memory.